modules/containers-signature-verify-enable.adoc

```diff
@@ -94,3 +95,91 @@ $ butane 51-master-rh-registry-trust.bu -o 51-master-rh-registry-trust.yaml
+99-master-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-master-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-master-ssh                                                                                 3.2.0             28m
+99-worker-generated-crio-seccomp-use-default                                                  3.2.0             25m
+99-worker-generated-registries                     a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             25m
+99-worker-ssh                                                                                 3.2.0             28m
+rendered-master-af1e7ff78da0a9c851bab4be2777773b   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             8s  <2>
+rendered-master-cd51fd0c47e91812bfef2765c52ec7e6   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-2b52f75684fbc711bd1652dd86fd0b82   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             24m
+rendered-worker-be3b3bce4f4aa52a62902304bac9da3c   a2178ad522c49ee330b0033bb5cb5ea132060b0a   3.2.0             48s
+----
+<1> New machine config
+<2> New rendered machine config 
+
+.. Check that the new machine config pool was created:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Sample output
+[source,terminal] 
+----
+NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
+master   rendered-master-af1e7ff78da0a9c851bab4be2777773b   True      False      False      3              3                   3                     0                      30m
+worker   rendered-worker-be3b3bce4f4aa52a62902304bac9da3c   False     True       False      3              0                   0                     0                      30m <1>
+----
+<1> The machine config pool is updating with the new machine config.
+
+. If your cluster uses any RHEL7 worker nodes, you need to create YAML files on those nodes in the `/etc/containers/registries.d` directory, which specify the location of the detached signatures for a given registry server:
+
+.. Start a debug session to each RHEL7 worker node:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+.. Change your root directory to the host:
++
+[source,terminal]
+----
+sh-4.2# chroot /host
+----
+
+.. Create the `/etc/containers/registries.d/registry.redhat.io.yaml` that contains the following:   
++
+[source,terminal]
+----
+docker:
+     registry.redhat.io:
+         sigstore: https://registry.redhat.io/containers/sigstore
+----
+
+.. Create the `/etc/containers/registries.d/registry.access.redhat.com.yaml` that contains the following:   
++
+[source,terminal]
+----
+docker:
+     registry.access.redhat.com:
+         sigstore: https://access.redhat.com/webassets/docker/content/sigstore
```

Comment:
is there a blurb saying this tutorial is assuming they're trying to access the RHEL images? this URL will be different for other images

Reply:
@haircommander The step starts with _If your cluster uses any RHEL7 worker nodes..._ as it appears that this issue occurs only with RHEL7 workers. 
@lyman9966 Can you confirm?

Reply:
@xiaojiey Can you help to confirm this point? Thanks

Reply:
AFAIK,the release registry are `registry.redhat.io` and `registry.access.redhat.com`, the sigstore for them are uniqe and fixed.
Please correct me if I'm wrong.

Reply:
correct, but my point is not about the rhel 7 nodes but the rhel images. for instance, if a user wanted like busybox to have signatures verified, then they'd need another blob in this file. We don't need to describe which or how, but I think it may be good to mention these only work for images hosted in registry.access.redhat.com

Reply:
@haircommander @xiuwang 

>  they'd need another blob in this file

Do you have an example blob for a different repo??

Also, I added _[The following example works only for images hosted in `registry.access.redhat.com`](https://github.com/openshift/openshift-docs/pull/45006/files#diff-dd5679be830cbcad0900cd578a973f49b91e207d1681cc0246b81ee9eed94296R140)._

Reply:
I think that blurb is fine, I don't have another example :smile: